packages/cli/package.json

```diff
@@ -48,6 +48,7 @@
     "colors": "1.3.3",
     "debug": "4.1.1",
     "fs-extra": "8.1.0",
+    "gulp-prettier": "3.0.0",
```

Comment:
why is this included? the regular `prettier` has a `format` we should be able to use.

Reply:
Good question! It's because Yeoman generator's `registerTransformStream` requires a vinyl object stream(like gulp) rather than a simple function: https://yeoman.io/authoring/file-system.html#transform-output-files-through-streams

Reply:
ah, makes sense. 

maybe a dumb question, but it doesn't look like we'll actually be feeding `prettier` in the generator any un-pretty files. the generated auth stuff is run through prettier, and presumably any templates we ship will have also been prettyified already. so i'm not sure this will actually do anything here.

Reply:
We'll still need run prettier for non-auth files, such as `package.json` and `index.js`. Since here the Yeoman generator runs prettier before writing any files, maybe we don't need to do any code formatting when generating auth stuff?

Reply:
_this comment was previously misplaced from an email, so i'm reposting it_

---

That's true. It looks like the index file is already pretty as it's stored here and package.json is easy to pretty print, but it's nbd either way. An extra dep doesn't hurt much.